<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CControllerWindowBase</class>
 <widget class="QWidget" name="CControllerWindowBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="typeGroupBox">
     <property name="title">
      <string>Type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="windowRadioButton">
        <property name="text">
         <string>Window</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="popupRadioButton">
        <property name="text">
         <string>Popup</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="dialogRadioButton">
        <property name="text">
         <string>Dialog</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="toolRadioButton">
        <property name="text">
         <string>Tool</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="sheetRadioButton">
        <property name="text">
         <string>Sheet</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="toolTipRadioButton">
        <property name="text">
         <string>Tooltip</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="drawerRadioButton">
        <property name="text">
         <string>Drawer</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="splashScreenRadioButton">
        <property name="text">
         <string>Splash screen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="hintsGroupBox">
     <property name="title">
      <string>Hints</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" rowspan="2">
       <widget class="QCheckBox" name="msWindowsFixedSizeDialogCheckBox">
        <property name="text">
         <string>MS Windows fixed size dialog</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="windowMaximizeButtonCheckBox">
        <property name="text">
         <string>Window maximize button</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QCheckBox" name="windowCloseButtonCheckBox">
        <property name="text">
         <string>Window close button</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="x11BypassWindowManagerCheckBox">
        <property name="text">
         <string>X11 bypass window manager</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="framelessWindowCheckBox">
        <property name="text">
         <string>Frameless window</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="windowContextHelpButtonCheckBox">
        <property name="text">
         <string>Window context help button</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="windowNoShadowCheckBox">
        <property name="text">
         <string>No drop shadow</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="windowShadeButtonCheckBox">
        <property name="text">
         <string>Window shade button</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="windowTitleCheckBox">
        <property name="text">
         <string>Window title</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="windowStaysOnTopCheckBox">
        <property name="text">
         <string>Window stays on top</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" rowspan="2">
       <widget class="QCheckBox" name="windowSystemMenuCheckBox">
        <property name="text">
         <string>Window system menu</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="windowStaysOnBottomCheckBox">
        <property name="text">
         <string>Window stays on bottom</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" rowspan="2">
       <widget class="QCheckBox" name="customizeWindowHintCheckBox">
        <property name="text">
         <string>Customize window</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="windowMinimizeButtonCheckBox">
        <property name="text">
         <string>Window minimize button</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>&amp;Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
